app/models/katello/alternate_content_source.rb

```diff
@@ -16,9 +16,22 @@ class AlternateContentSource < Katello::Model
 
     encrypts :upstream_password
 
+    # Validate the ssl_id params.
```

Comment:
Relationships aren't technically validations; I'd remove this comment

Reply:
Done!